README.md

```diff
@@ -55,11 +61,38 @@ Install Cachix from master:
     $ nix-env -if https://github.com/cachix/cachix/tarball/master --substituters 'https://cache.nixos.org https://cachix.cachix.org' --trusted-public-keys 'cachix.cachix.org-1:eWNHQldwUO7G2VkjpnjDbWwy4KQ/HNxht7H4SSoMckM= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY='
 ```
 
-Or with Nix 2.4+:
 
-```
-    $ nix profile install github:cachix/cachix/latest
-```
+### Dev setup for VS Code
+
+1. [Enable flakes](https://nixos.wiki/wiki/Flakes#Enable_flakes)
+
+2. Install [direnv](https://direnv.net/)
+
+3. Clone this repo
+  ```sh
+  git clone https://github.com/br4ch1st0chr0n3/cachix
+  cd cachix
+  ```
+
+4. Try using cache
+  1. Install [cachix](https://docs.cachix.org/installation)
+  2. Pull cache
+  ```sh
+  cachix use cachix-shell
```

Comment:
cachix-shell?

Reply:
I created a cache for my `x86_64-linux` system